config/environments/test.rb

```diff
@@ -5,7 +5,7 @@
   # test suite. You never need to work with it otherwise. Remember that
   # your test database is "scratch space" for the test suite and is wiped
   # and recreated between test runs. Don't rely on the data there!
-  config.cache_classes = true
+  config.cache_classes = false
```

Comment:
Here's what we like to stick in `test.rb`:

```ruby
  config.cache_classes = ActiveRecord::Type::Boolean.new.cast(ENV.fetch('CACHE_CLASSES', '1'))
  config.eager_load = ActiveRecord::Type::Boolean.new.cast(ENV.fetch('EAGER_LOAD', '0'))
  config.assets.compile = ActiveRecord::Type::Boolean.new.cast(ENV.fetch('COMPILE_ASSETS', '1'))
```

This gives us finer-grained controls for running tests locally (where we don't want to eager load all classes) vs in CI (where we _do_ want to fully eager load everything, and also precompile and test with production-like asset behaviors).

Reply:
Similarly, here's what we could probably stick in `development.rb`:

```ruby
  config.cache_classes = ActiveRecord::Type::Boolean.new.cast(ENV.fetch('CACHE_CLASSES', '0'))
  config.eager_load = ActiveRecord::Type::Boolean.new.cast(ENV.fetch('EAGER_LOAD', '0'))
  config.assets.compile = ActiveRecord::Type::Boolean.new.cast(ENV.fetch('COMPILE_ASSETS', '1'))
```

Reply:
I could see a case for having `'0'` for `CACHE_CLASSES` in test.rb, if that's the new default.